docs/proposals/maturity.md

```diff
@@ -0,0 +1,242 @@
+# Summary and Problem Definition:
+
+This document is a proposal to enhance operator capability definitions in order to be able to assess or audit automatically what level requirement they satisfy with their features. 
+
+We have another document that was created before for automated audits https://docs.google.com/document/d/14nK3krb_cctL8cfk1Fwu1dkAEfo9ZC_iaGCYN4eKMmY/edit# . The goal here are the level definitions. Not the automation tooling in the sense of implementation. That's another track that is also being treated by our team. The information defined here will feed the directions and implementations on the opcap tools on running capability audits against the operators and their operands.
+
+Two main key principles are supposed to be present when evaluating those capabilities. 
```

Comment:
Are these principles the next 2 titled sections? It's unclear since it's not specified and all the headings are top level headings.